### PR TITLE
change torch.equal() to self.assetrEqual() while comparing NHWC and NCHW batchnorm output

### PR DESCRIPTION
**Pull request by dnikolaev-amd**
`self.assertTrue(torch.equal(out1, out2))` assumes a compete match
But we have a slight difference (~1e-7) with fp32 NHWC and NCHW batchnorm output
`self.assertEqual(out1, out2)` allows for tolerance

### CODE REVIEW

**Comment by rocm-mici** [comment]
Jenkins build for [d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6](https://github.com/ROCm/pytorch/commit/d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6) commit finished as FAILURE
Links: [Blue Ocean view](http://ml-ci-internal.amd.com:8080/blue/organizations/jenkins/pytorch%2Fpytorch-ci-pipeline/detail/PR-1600/3//pipeline) / [Build artifacts](http://ml-ci-internal.amd.com:8080/job/pytorch/job/pytorch-ci-pipeline/job/PR-1600/3/artifact)

**Comment by rocm-mici** [comment]
Jenkins build for [d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6](https://github.com/ROCm/pytorch/commit/d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6) commit finished as FAILURE
Links: [Blue Ocean view](http://ml-ci-internal.amd.com:8080/blue/organizations/jenkins/pytorch%2Fpytorch-ci-pipeline/detail/PR-1600/4//pipeline) / [Build artifacts](http://ml-ci-internal.amd.com:8080/job/pytorch/job/pytorch-ci-pipeline/job/PR-1600/4/artifact)

**Comment by rocm-mici** [comment]
Jenkins build for [d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6](https://github.com/ROCm/pytorch/commit/d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6) commit finished as FAILURE
Links: [Blue Ocean view](http://ml-ci-internal.amd.com:8080/blue/organizations/jenkins/pytorch%2Fpytorch-ci-pipeline/detail/PR-1600/5//pipeline) / [Build artifacts](http://ml-ci-internal.amd.com:8080/job/pytorch/job/pytorch-ci-pipeline/job/PR-1600/5/artifact)

**Comment by rocm-mici** [comment]
Jenkins build for [d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6](https://github.com/ROCm/pytorch/commit/d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6) commit finished as FAILURE
Links: [Blue Ocean view](http://ml-ci-internal.amd.com:8080/blue/organizations/jenkins/pytorch%2Fpytorch-ci-pipeline/detail/PR-1600/6//pipeline) / [Build artifacts](http://ml-ci-internal.amd.com:8080/job/pytorch/job/pytorch-ci-pipeline/job/PR-1600/6/artifact)

**Comment by rocm-mici** [comment]
Jenkins build for [d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6](https://github.com/ROCm/pytorch/commit/d6ab6a27f6ea46bf9ba3c62e93f128716f09a5f6) commit finished as FAILURE
Links: [Blue Ocean view](http://ml-ci-internal.amd.com:8080/blue/organizations/jenkins/pytorch%2Fpytorch-ci-pipeline/detail/PR-1600/7//pipeline) / [Build artifacts](http://ml-ci-internal.amd.com:8080/job/pytorch/job/pytorch-ci-pipeline/job/PR-1600/7/artifact)

**Review by pruthvistony — APPROVED** [review]
